Standards/scs-XXXX-vN-Availability-Zones-Standard.md

```diff
@@ -0,0 +1,196 @@
+| Storage            | A generic name for the IaaS service, that manages the storage backends and virtual devices (e.g. Cinder in OpenStack).                   |
+
+## Motivation
+
+Redundancy is a non-trivial but relevant issue for a cloud deployment.
+First and foremost it is necessary to increase failure safety through redundancy on the physical layer.
+The IaaS layer as the first abstraction layer from the hardware has an important role in this topic, too.
+The grouping of redundant physical resources into Availability Zones on the IaaS level, gives customers the option to distribute their workload to different AZs which will result in a better failure safety.
+While CSPs already have some similarities in their grouping of physical resources to AZs, there are also differences.
+This standard aims to reduce thos differences and will clarify, what customers can expect from Availability Zones in IaaS.
+
+Availability Zones in IaaS can be set up for Compute, Network and Storage while all refering to the same physical separation in a deployment.
+This standard elaborates the necessity of having Availability Zones for each of these classes.
+It will also check the requirements customers may have, when thinking about Availability Zones in relation to the taxonomy of failure safety levels [^1].
+The result should enable CSPs to know when to create AZs to be SCS-compliant.
+
+## Design Considerations
+
+Availability Zones should represent parts of the same physical deployment that are independent of each other.
+The maximum of physical independence is achieved through putting physical machines into different fire zones.
+In that case a failure case up to level 3 as described in the taxonomy of failure safety levels document[^1] will not lead to a complete outage of the deployment.
+
+Having Availability Zones represent fire zones will also result in AZs being able to take workload from another AZ in a Failure Case of Level 3.
+So that even the destruction of one Availability Zone will not automatically include the destruction of the other AZs.
+
+:::caution
+
+Even with fire zones being physically designed to protect parts of a data center from severe destruction in case of a fire, this will not always succeed.
+Availability Zones in Clouds are most of the time within the same physical data center.
+In case of a big catastrophe like a huge fire or a flood the whole data center could be destroyed.
+Availability Zones will not protect customers against these failure cases of level 4 of the taxonomy of failure safety[^1].
+
+:::
+
+Smaller deplyoments like edge deployments may not have more than one fire zone in a single location.
+To include such deployments, it should not be required to use Availability Zones.
+
+Other physical factors that should be considered are the power supplies, internet connection, cooling and core routing.
+Availability Zones have been also being configured to show redundancy in e.g. Power Supply as in the PDU.
```

Comment:
There seems to be something wrong with this sentence. I don't understand it.

Reply:
I changed the whole paragraph, please re-read it.